### PR TITLE
issues #226, #227, #155

### LINKED ISSUE #155: rankings display fixes
http://beta.opendota.com/players/88367253/rankings

- [x] last played always shows as "never".  I think this is due to not returning the last played date on this endpoint.  We should have an option on the hero cell to hide this value.
- [x] We should make the rank an ordinal number.  There's a utility function in core to do this (or just Google for it)

- [x] Current UI has context coloring based on the percentile (green/blue/dark blue/yellow/red).  We can add this to the new UI.

- [x] Hero should link to rankings page

- [x] Rank column should only add the english in the display function. Otherwise sorting is as a string and doesn't sort properly.

### LINKED ISSUE #226: Players/Pros tabs should use combined cell
Similar to how the hero table works, the player and last played time should be combined and the sort should be on the last played time.

Pros should also display image (I think backend has been updated for this)

### LINKED ISSUE #227: Records should use combined cell
I think the Set column can be removed and combined with the hero cell.  We can make the record value column link to the match instead?

### PR DESCRIPTION
**Pull request by blukai**
fixes #226 
fixes #227 
fixes #155
image

### CODE REVIEW

File: src/lang/en.json [hunk 1]
```diff
@@ -320,6 +320,7 @@
   "tooltip_biggest_hit": "Largest instance of damage on a hero",
   "tooltip_damage_dealt": "Damage dealt to heroes by items/abilities",
   "tooltip_damage_received": "Damage received from heroes by items/abilities",
+  "tooltip_registered_user": "Registered user",
```
Comment (blukai) [inline src/lang/en.json]: Ye, but not now, 🍀  I'll plan to make some changes to player header that fixes this https://github.com/odota/ui/issues/81#issuecomment-251370349. Then I will add strings..

**Comment by blukai** [comment]
fixes #227 
image

**Comment by blukai** [comment]
almost #155
image

https://github.com/odota/ui/issues/155#issuecomment-252042343

File: src/lang/en.json [hunk 2]
```diff
@@ -53,6 +53,9 @@
   "error": "Error",
   "error_message": "Whoops! Something went wrong.",
   "error_four_oh_four_message": "The page you are looking for cannot be found.",
+
+  "unknown": "Unknown",
```
Comment (howardchung) [inline src/lang/en.json]: these strings should probably be namespaced under `general_` and moved into alphabetical order (by namespace).

File: src/lang/en.json [hunk 3]
```diff
@@ -335,5 +339,8 @@
   "teamfight_score": "Radiant kills/Dire kills",
   "teamfight_radiant_gold_adv": "Net gold advantage for Radiant",
   "teamfight_radiant_xp_adv": "Net XP advantage for Radiant",
-  "anonymous": "Anonymous"
+  "anonymous": "Anonymous",
+
+  "radiant": "Radiant",
```
Comment (howardchung) [inline src/lang/en.json]: same with these strings

File: src/utility/index.jsx [hunk 4]
```diff
@@ -124,30 +118,67 @@ export const getOrdinal = (n) => {
...
 const getSubtext = (row) => {
   if (row.match_id && row.player_slot !== undefined) {
-    // TODO localize
-    return isRadiant(row.player_slot) ? 'Radiant' : 'Dire';
+    return isRadiant(row.player_slot) ? strings.radiant : strings.dire;
   }
   if (row.last_played) {
     return <FromNowTooltip timestamp={row.last_played} />;
   }
+  if ((row.name && row.value) !== undefined) {
```
Comment (howardchung) [inline src/utility/index.jsx]: why not just check start_time?  If you don't want to render any subtext, that should be determined by the parent component.

File: src/utility/index.jsx [hunk 5]
```diff
@@ -124,30 +118,67 @@ export const getOrdinal = (n) => {
...
+    return <FromNowTooltip timestamp={row.start_time} />;
+  }
   return null;
 };
 
+const percentColor = (pct) => {
```
Comment (howardchung) [inline src/utility/index.jsx]: If we are using the function for multiple types of bucketing, maybe give it a more generic name (without "Color").

For analysis, we currently use letter grades, but I like the sentiment google material icons:
https://design.google.com/icons/

Comment (blukai) [inline src/utility/index.jsx]: Hey, we can use dota emoticons instead if letters or gm icons. I'll do it after a while 😺

File: src/components/Visualizations/Table/Player.jsx [hunk 6]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
```
Comment (blukai) [inline src/components/Visualizations/Table/Player.jsx]: I think yes. 🍀

**Comment by blukai** [comment]
Not yet ready for merge 🐰

**Comment by howardchung** [comment]
ready for merge?

**Comment by howardchung** [comment]
This LGTM, I'll merge (if there were any other changes you wanted to make, please open a new PR)

**Pull request merged**